examples/notebooks/rag/requirements.txt

```
@@ -2,7 +2,7 @@
 
 data-prep-toolkit-transforms==0.2.1
 data-prep-toolkit-transforms-ray==0.2.1
-
+data-prep-connector==0.2.2.dev1
```

We may want to use the official release version.

```suggestion
data-prep-connector==0.2.2
```